silx/gui/plot3d/SFViewParamTree.py

```diff
@@ -155,7 +155,7 @@ def slotfn(*args, **kwargs):
                 for sigIdx, signal in enumerate(signals):
                     slot = gen_slot(sigIdx)
                     signal.connect(slot)
-                    slots.append((signal, slot))
+                    slots.append(slot)
```

Comment:
`self.__slots` is used in `_disconnectSignals` which unpacks `signal` and `slot`... but `_disconnectSignals` doesn't seem to be used anywhere, so it can be removed...